src/components/Menu/Menu.test.tsx

```diff
@@ -58,14 +58,14 @@ describe('Menu Component', () => {
     await waitFor(() => expect(asFragment()).toMatchSnapshot())
   })
 
-  test('renders primary menu correctly', () => {
+  test('renders primary menu correctly', async() => {
     const { asFragment } = render(<Menu hierarchy={Primary} items={items} />)
-    expect(asFragment()).toMatchSnapshot()
+    await waitFor(() => expect(asFragment()).toMatchSnapshot())
```

Comment:
?

Reply:
If not specified, this error is logged in console: 
`Warning: An update to ForwardRef(StickyScrollBar) inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):`